app/components/work_packages/date_picker/date_form_component.html.erb

```diff
@@ -18,7 +18,7 @@
         end
 
         start_date.with_row(classes: "wp-datepicker-dialog-date-form--text-field-container") do
-          render(Primer::Alpha::TextField.new(**text_field_options(name: :start_date, label: start_date_label), aria: { live: :polite }))
+          render(Primer::Alpha::TextField.new(**text_field_options(name: :start_date, label: start_date_label), aria: { live: :polite, atomic: true }))
```

Comment:
Since the aria attributes are the same for each text field, it probably makes sense to add them to the `text_field_options` probably even to the `default_field_options` in the class.